CONTRIBUTING.md

```diff
@@ -100,7 +100,19 @@ Android Studio](https://developer.android.com/studio/run/managing-avds). Choose
 
 ### Starting the dev version of Mapeo Mobile
 
-Connect your phone with USB, or start up the emulator, then build and run the
+Build translations with:
+
+```sh
+npm run build:translations
+```
+
+Connect your phone with USB, or start up the emulator. Then start the Javascript bundler:
+
+```sh
+npm start
```

Comment:
The `npm run android` command should open a new terminal with the bundler if it is not already running, however that might not work on all machines, so I agree with your proposed changes here, which are more reliable.

Reply:
Yea, doesn't do that on my machine.